client/app/components/compose.coffee

```diff
@@ -311,9 +312,13 @@ module.exports = Compose = React.createClass
 
         # new draft
         else
-            state = messageUtils.makeReplyMessage @props.selectedAccountLogin,
-                @props.inReplyTo, @props.action,
-                @props.settings.get('composeInHTML')
+            state = messageUtils.makeReplyMessage(
+                @props.selectedAccountLogin,
+                @props.inReplyTo,
+                @props.action,
+                @props.settings.get('composeInHTML'),
+                @props.selectedAccount.get('signature')
```

Comment:
This serie of components get passed the selectedAccountID, all the accounts, the selectedAccountLogin and now the selectedAccount itself.

I am not sure which system is better, but we really ought to decide 

EIther we pass the `selectedAccountID` & `accounts` and then do `accounts.get(selectedAccountID).get('login')` and `accounts.get(selectedAccountID).get('conversation')`

Or we  pass the selectedAccount itself and get the ID from it.

Ping people working more on the frontend @clochix and @jsilvestre  ?